generate/post_process_json.go

```diff
@@ -0,0 +1,24 @@
+package generate
+
+import (
+	"bufio"
+	"bytes"
+	"regexp"
+)
+
+// jsonTagRegexp is the regular express to make the json struct tag to the same name as the one defined in protobuf tag.
+// Given this input: `protobuf:"bytes,1,opt,name=FirstName,json=first_name,proto3" json:"first_name,omitempty"`
+// The capture groups are:
+// * $1: `protobuf:"bytes,1,opt,name=FirstName,json=first_name,proto3" json:"
+// * $2: first_name
+// * $3: FirstName
+// * $4: ,omitempty"`
+var jsonTagRegexp = regexp.MustCompile("(`" + `protobuf:"bytes,\d+,opt,name=[\w_]+,json=([\w_]+),proto3" json:")([\w+]+)(,omitempty"` + "`)")
```

Comment:
just a thought - maybe using https://pkg.go.dev/github.com/fatih/structtag?tab=doc#Parse is more robust and maintanable?

Reply:
maybe, @mvdan your thought?

Reply:
Yes, using regex here is... not good.

The simplest alternative would be https://golang.org/pkg/reflect/#StructTag. Note that it doesn't support splitting the value into many comma-separated values, but that's trivial to do via `strings.Split`. I don't think we need to go as far as an external dependency for that.